style.css

```diff
@@ -0,0 +1,67 @@
+.headline {
+  background-image: url("./images/header-illsutration-mobile.svg");
+  background-repeat: no-repeat;
+  background-size: 100%;
+  position: absolute;
+}
+
+.personal-presentation h1 {
+  font-family: 'Inter', sans-serif;
+  font-weight: 700;
+  font-size: 40px;
+  text-align: center;
+  line-height: 52px;
+  font-style: normal;
+  margin-top: 58px;
+  margin-left: 27px;
+  margin-right: 27px;
+  margin-bottom: 0%;
+}
+
+.personal-presentation p {
+  font-family: 'Inter', sans-serif;
+  font-weight: 400;
+  font-size: 19px;
+  text-align: center;
+  line-height: 28px;
+  font-style: normal;
+  margin-left: 27px;
+  margin-right: 27px;
+  margin-top: 15px;
+}
+
+.social-media ul {
+  list-style: none;
+}
+
+.color-blue {
+  color: #172b4d;
+}
+
+#title-green {
+  color: #36b37e;
+}
+
+nav {
+  display: flex;
+  justify-content: end;
+}
+
+nav img {
+  margin-right: 38px;
+  margin-top: 40px;
+}
+
+.social-media-links {
+  display: flex;
+  justify-content: left;
+  gap: 5%;
+  padding-left: 100px;
```

Comment:
- [ ] I studied your reason for doing this like this, by using justify-content: left; and padding-left 100px which isn't responsive.

<img width="155" alt="image" src="https://github.com/AleMdta/portfolio-mobile-version/assets/41163571/41ca2510-56b4-4780-8f19-48f6b5294837">

- After adding border to the container, removing the margin and centering the element I realized there is some left margin outside the container
<img width="175" alt="image" src="https://github.com/AleMdta/portfolio-mobile-version/assets/41163571/5ac5002a-cbd6-4654-bbe2-b35386204812">

On default the browser adds default margin and border to elements and containers, to get rid of this action you'll have to reset default styles, adding this to your styles.
```
*,
*::before,
*::after {
  margin: 0;
  padding: 0;
  box-sizing: border-box;
}
```
This resets the styles
<img width="212" alt="image" src="https://github.com/AleMdta/portfolio-mobile-version/assets/41163571/bc4166b8-8f8c-4aa1-b01d-16504e2603a6">
 
You can see the default margin has been removed.

- [ ] Lastly, you should align the item properly horizontally by shifting the M icon a bit, and probably making it bigger, try thing and tweak things like a developer will do, have a nice time 👋